### ISSUE TITLE
PyTorch support

### ISSUE THREAD

**Issue by shubhamjain0594** [marcoancona/DeepExplain#1]
I am interested in developing pytorch support. Any channel where that discussions are going on?

I have background in PyTorch, active contributor to visdom, and other libraries.

**Comment by rohitghosh** [comment]
+ 1 @shubhamjain0594. Would love to add support for Pytorch @marcoancona - any development that we need to be aware of?

**Comment by marcoancona** [comment]
Hi all,
although we have planned support for PyTorch, we have not started implementing it nor gave a lot of thought about possible technical challenges. However, gradient-override in PyTorch should be easier than in Tensorflow.
The idea is that PyTorch should be implemented in a module `deepexplain.pytorch` and should expose the same API. Then it would be used the same way, except from for the `import` statement:
```
from deepexplain.tensorflow import DeepExplain
from deepexplain.pytorch import DeepExplain
```
Pull requests are welcome. If you have in mind a possible discussion channel that could be opened for this let me know.
